projects/plugins/jetpack/extensions/blocks/seo/index.js

```diff
@@ -13,24 +14,28 @@ export const name = 'seo';
 export const settings = {
 	render: function JetpackSEODescriptionPanel() {
```

Comment:
Since this panel now contains the title and description fields, should we rename this function for clarity?